eng/sendtohelix.proj

```diff
@@ -80,11 +80,24 @@
     <_RuntimeInputs Include="$(TestHostRootPath)**/*.dll" />
   </ItemGroup>
 
+  <!-- Add global tools to runtime -->
+  <ItemGroup Condition="'$(TargetsNetCoreApp)' == 'true'">
+    <TestArchiveRuntimeDependencies Include="$(GlobalToolsDir)**\*.*" />
```

Comment:
Nit: prefix with _?

Reply:
This can be used from outside to add additional runtime dependencies therefore no underscore.